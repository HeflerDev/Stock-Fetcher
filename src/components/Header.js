import React from 'react';

const Header = () => (
  <header>
    <div className="header-img stack">
      <div className="header-text-container">
        <p>Progress</p>
        <h1>
          Empowering
          <br />
          Our Clients
        </h1>
        <p className="legend">
          We have over 20 years of consultancy experience in this area of business
          {' '}
          and we strive to achieve the following: develop, empower, enable and maintain.
        </p>
      </div>
    </div>
  </header>
);

export default Header;
