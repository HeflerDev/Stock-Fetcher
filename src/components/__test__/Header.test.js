import { configure, shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import renderer from 'react-test-renderer';
import Header from '../Header';

configure({ adapter: new Adapter() });

describe('when loading page', () => {
  const wrapper = shallow(<Header />);
  const tree = renderer
    .create(<Header />)
    .toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });

  it('has header', () => {
    expect(wrapper.contains(<p>Progress</p>)).toEqual(true);
  });
});
