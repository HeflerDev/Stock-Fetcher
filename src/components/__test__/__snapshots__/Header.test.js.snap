// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when loading page renders correctly 1`] = `
<header>
  <div
    className="header-img stack"
  >
    <div
      className="header-text-container"
    >
      <p>
        Progress
      </p>
      <h1>
        Empowering
        <br />
        Our Clients
      </h1>
      <p
        className="legend"
      >
        We have over 20 years of consultancy experience in this area of business
         
        and we strive to achieve the following: develop, empower, enable and maintain.
      </p>
    </div>
  </div>
</header>
`;
